src/styles/main.scss

```diff
@@ -0,0 +1,75 @@
+$item-bg-color: #eee;
+$calendar-item-size: 100px;
+$calendar-gap: 1px;
+$calendar-item-width: $calendar-item-size + $calendar-gap;
+$week: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px;
+  gap: $calendar-gap;
+  width: calc($calendar-item-width * 7);
```

Comment:
The width of the `.calendar` is calculated with a hardcoded value of 7 (days in a week). If the number of days in a week were to change or be configurable, this would lead to incorrect widths. Consider using a variable or a calculation based on the number of days.